direct/data/transforms.py

```diff
@@ -34,7 +42,7 @@ def to_tensor(data: np.ndarray) -> torch.Tensor:
     return torch.from_numpy(data)
 
 
-def verify_fft_dtype_possible(data: torch.Tensor, dims: Tuple[int, ...]) -> bool:
+def verify_fft_dtype_possible(data: torch.Tensor, dims: tuple[int, ...]) -> bool:
     """fft and ifft can only be performed on GPU in float16 if the shapes are powers of 2. This function verifies if
```

Comment:
I think you can make this tuple[int, int] | tuple[int, int, int]? You can define that type somewhere else and use it here

Reply:
Done. I didnt define it as a new type only using it there for now.